.github/workflows/test.yml

```diff
@@ -195,13 +195,6 @@ jobs:
           command: build
           args: --tests --all-features ${{ env.CARGO_FLAGS }}
 
-      - name: Build explorer
```

Comment:
The explorer should be built anyway with cargo test right?

Reply:
The lib should, since it's going to be used.